### PR TITLE
solution

### PR DESCRIPTION
**Pull request by abcdesign1**

### CODE REVIEW

**Review by VadKol — CHANGES_REQUESTED** [review]
Write more readable code

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days < 3
+    ? days * 40
+    : days >= 3 && days < 7
+      ? (days * 40) - 20
+      : (days * 40) - 50;
```
[the remark this answers is not in the document]
Comment (vadym4che) [reply src/calculateRentalCost.js]: explaination for each of the code are provided

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days < 3 // if car is rented for less then 3 days
+    ? days * 40 // total cost will be equal to clear cost-per-day times days
+    : days >= 3 && days < 7 // othrwise we'll check if term is between 3 and 6
+      ? (days * 40) - 20 // in this case total cost will be decreased on 20
+      : (days * 40) - 50; // last case - `days` is 7+ - we'll get -$50
```
Comment (illiarerun) [inline src/calculateRentalCost.js:35]: nested ternaries are hard to read, even if they are well-documented. best documentation is well-written code, so please remake it with plain if's

Comment (illiarerun) [inline src/calculateRentalCost.js:35]: [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 40, 50, 20, 7 and 3 are

Comment (illiarerun) [reply src/calculateRentalCost.js:35]: you repeat `days * 40` calculation 3 times, create a variable for it

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 2 above)
Comment (vadym4che) [reply src/calculateRentalCost.js]: fixed all issues

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,25 @@
...
-  // write code here
+  const dayCost = 40;
+  const threeDayBonus = 20;
+  const sevenDayBonus = 50;
+  const costWithoutBonus = days * dayCost;
+  let totalCost;
+
+  if (days < 3) {
+    totalCost = costWithoutBonus;
+  }
```
Comment (sergii-nosachenko) [inline src/calculateRentalCost.js:39]: Useless condition

File: src/calculateRentalCost.js [hunk 4]
```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const threeDayBonus = 20;
+  const sevenDayBonus = 50;
+  const costWithoutBonus = days * dayCost;
+  let totalCost;
```
Comment (sergii-nosachenko) [inline src/calculateRentalCost.js:35]:
```suggestion
  let totalCost = days * dayCost;
```

File: src/calculateRentalCost.js [hunk 5]
```diff
@@ -28,7 +28,25 @@
...
+
+  if (days < 3) {
+    totalCost = costWithoutBonus;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalCost = costWithoutBonus - threeDayBonus;
```
Comment (sergii-nosachenko) [inline src/calculateRentalCost.js:42]: Use `-=` to adjust final price

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]
A quick fix is needed

File: src/calculateRentalCost.js (hunk 3 above)
Comment (vadym4che) [reply src/calculateRentalCost.js]: fixed + assigned total as default with ability to be changed

File: src/calculateRentalCost.js (hunk 4 above)
Comment (vadym4che) [reply src/calculateRentalCost.js]: fixed

File: src/calculateRentalCost.js (hunk 5 above)
Comment (vadym4che) [reply src/calculateRentalCost.js]: shortened

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**